docs/how-to/instrumenting-rewriting-binary-application.rst

```diff
@@ -0,0 +1,942 @@
+.. meta::
+   :description: Omnitrace documentation and reference
+   :keywords: Omnitrace, ROCm, profiler, tracking, visualization, tool, Instinct, accelerator, AMD
+
+****************************************************
+Instrumenting and rewriting a binary application
+****************************************************
+
+There are three ways to perform instrumentation with the ``omnitrace-instrument`` executable:
```

Comment:
@dgaliffiAMD I reworked your suggested edit into a list and slightly modified the main bullet points below. Please let me know if you think more changes are required.